### PR TITLE
Update to readme to include Open MPI compile/install instructions

### PR DESCRIPTION
**Pull request by manjugv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,21 +43,53 @@ UCC is BSD-style licensed, as found in the [LICENSE](LICENSE) file.
...
+```sh
+$ git clone https://github.com/open-mpi/ompi
+$ cd ompi
+$ ./autogen.pl; ./configure --prefix=<ompi-install-path> --with-ucx=<ucx-install-path> --with-ucc=<ucc-install-path>; make -j install
+```
+
+### Run MPI programs
+
+```sh
+mpirun -np 2 --mca coll_ucc_enable 1 --mca coll_ucc_priority 100 ./my_mpi_app
```
Comment (Sergei-Lebedev) [inline README.md:84]: add "$ " before mpirun

File: README.md [hunk 2]
```diff
@@ -43,21 +43,53 @@ UCC is BSD-style licensed, as found in the [LICENSE](LICENSE) file.
...
+```
+
+### Run OpenSHMEM programs
+
+```sh
+mpirun -np 2 --mca scoll_ucc_enable 1 --mca scoll_ucc_priority 100 ./my_openshmem_app
```
Comment (Sergei-Lebedev) [inline README.md:90]: add "$ " before mpirun

File: README.md [hunk 3]
```diff
@@ -43,21 +43,53 @@ UCC is BSD-style licensed, as found in the [LICENSE](LICENSE) file.
...
 $./autogen.sh
-
 $./configure --prefix=<ucc-install-path> --with-docs-only
-
 $make docs
 ```
 
+### Open MPI and UCC collectives
+
+#### Compile UCX
```
Comment (Sergei-Lebedev) [inline README.md:59]: is it intentional to have smaller size header for compile section vs run section?

File: README.md [hunk 4]
```diff
@@ -43,21 +43,53 @@ UCC is BSD-style licensed, as found in the [LICENSE](LICENSE) file.
 ### Developer's Build 
 ```sh
 $./autogen.sh
-
 $./configure --prefix=<ucc-install-path> --with-ucx=<ucx-install-path>
```
Comment (Sergei-Lebedev) [inline README.md:46]: no space between "$" and command, maybe add it since changing readme file anyway?

**Review by Sergei-Lebedev — APPROVED** [review]

**Review by vspetrov — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (manjugv) [reply README.md:59]: It was, but it doesn't make sense.

**Pull request merged**